cluster/gce/util.sh

```diff
@@ -1580,7 +1580,7 @@ CNI_CONFIG_DIR: $(yaml-quote "${WINDOWS_CNI_CONFIG_DIR}")
 WINDOWS_CNI_STORAGE_PATH: $(yaml-quote "${WINDOWS_CNI_STORAGE_PATH}")
 WINDOWS_CNI_VERSION: $(yaml-quote "${WINDOWS_CNI_VERSION}")
 WINDOWS_CONTAINER_RUNTIME: $(yaml-quote "${WINDOWS_CONTAINER_RUNTIME}")
-WINDOWS_CONTAINER_RUNTIME_ENDPOINT: $(yaml-quote "${WINDOWS_CONTAINER_RUNTIME_ENDPOINT}")
+WINDOWS_CONTAINER_RUNTIME_ENDPOINT: $(yaml-quote "${WINDOWS_CONTAINER_RUNTIME_ENDPOINT:-}")
```

Comment:
hmm why is only this one needing this and not the others? probably should have a default like the others instead?

Reply:
it is initialised inside an if 
```
 if [[ "${WINDOWS_CONTAINER_RUNTIME:-}" != "docker" ]]; then
    flags+=" --container-runtime=remote"
    if [[ "${WINDOWS_CONTAINER_RUNTIME}" == "containerd" ]]; then
      WINDOWS_CONTAINER_RUNTIME_ENDPOINT=${KUBE_WINDOWS_CONTAINER_RUNTIME_ENDPOINT:-npipe:////./pipe/containerd-containerd}
      flags+=" --container-runtime-endpoint=${WINDOWS_CONTAINER_RUNTIME_ENDPOINT}"
    fi
  fi

```

Reply:
IIRC this should come from config-default.sh or config-test.sh

cc @amwat (extra eyes on these scripts)

Reply:
the file generated in line 1336 is defaulting this way too
https://github.com/kubernetes/kubernetes/pull/99990/files#diff-09816a4dd2fd3cf28f97bf9f34330a66336c631830b103d0c96d71eb632dab0dR1336-R1357

Reply:
yeah the actual value should ideally come from `config-default.sh` or `config-test.sh`
but empty fallback at this point sgtm, probably should have it for all others as well (just in case).